src/firebase.js

```diff
@@ -1,5 +1,6 @@
 import * as firebase from "firebase/app";
 import * as authObject from "firebase/auth";
+import { getDatabase } from "firebase/database";
```

Comment:
See if you can use the functions on here to call the rewards data: src/functions/partnerData.js